std/index.ts

```diff
@@ -27,3 +27,4 @@ export { Encoding, read } from './read';
 export { info, dir } from './fs';
 export { param } from './param';
 export { mix, patch, merge } from './merge';
+export { parse, unparse } from './parse';
```

Comment:
Should we have a `@jkcfg/format` module instead of adding it to the std lib? (the std re-exports for fs, param, merge are going away in `0.3.0`)

Reply:
Yeah could do -- if I did it as `YAML.parse` etc., I'd put it in format.ts and it could be imported with

    import { YAML } from '@jkcfg/std/format';

which is neater than having to import from two places to be able to use it.